pkgs/development/compilers/gcc/common/builder.nix

```diff
@@ -313,6 +311,9 @@ originalAttrs:
       # More dependencies with the previous gcc or some libs (gccbug stores the build command line)
       rm -rf $out/bin/gccbug
 
+      # Remove .la files, they're incorrectly generated and confuse libtool
```

Comment:
Is this an upstream GCC problem, or because we're moving stuff around?  (Incorrectly generated implies the former.)

Reply:
Updated the comment.